xdsl/dialects/memref.py

```diff
@@ -263,10 +263,12 @@ def verify_(self) -> None:
                             "dense type or an unit attribute")
 
     @staticmethod
-    def get(sym_name: str | StringAttr,
-            typ: Attribute,
-            initial_value: Optional[Attribute],
-            sym_visibility: str = "private") -> Global:
+    def get(
+        sym_name: StringAttr,
```

Comment:
As far as I understand, the str was no longer supported, and this was not caught due to lack of annotations